geniza/corpus/templates/corpus/snippets/document_option_label.html

```diff
@@ -0,0 +1,25 @@
+{# template snippet for displaying document label on a form #}
+{# used on document merge form to provide enough information to merge accurately #}
+<div class="merge-document-label">
+    <strong><a target="_blank" href="{% url 'admin:corpus_document_change' document.id %}">{{ document }}</a></strong>
```

Comment:
I wanted to click the document title to select the primary, so I didn't like the fact that it was linked (although linking to the change form is helpful!). Let's follow django admin convention and display a pencil edit icon to the left of the document id as the link to the document change form.